LibreHardwareMonitorLib/Hardware/Motherboard/Identification.cs

```diff
@@ -411,6 +411,8 @@ public static Model GetModel(string name)
                 return Model.B660M_DS3H_AX_DDR4;
             case var _ when name.Equals("ROG STRIX Z790-I GAMING WIFI", StringComparison.OrdinalIgnoreCase):
                 return Model.ROG_STRIX_Z790_I_GAMING_WIFI;
+            case var _ when name.Equals("MPG X570 GAMING PLUS (MS-7C37)",StringComparison.OrdinalIgnoreCase):
```

Comment:
Please move this added motherboard to a separate PR.

Reply:
oups, sorry about that it was not supposed to be here